py/torch_tensorrt/dynamo/conversion/_conversion.py

```diff
@@ -107,6 +107,36 @@ def interpret_module_to_result(
         compilation_settings=settings,
     )
     interpreter_result = interpreter.run()
+
+    if settings.make_refitable:
```

Comment:
Wouldnt engine caching require refit to be on?

Reply:
> Wouldnt engine caching require refit to be on?

yep. it requires make_refitable == True.